Examples/DynamicColorExample.xcodeproj/project.pbxproj

```diff
@@ -1300,8 +1301,9 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = J87V2D9N4D;
```

Comment:
this shouldn't be here

Reply:
Thanks @andrewSB - new commit resetting it to "none" has been pushed.